extensions/sql-database-projects/images/sqlDatabaseProjects.svg

```diff
@@ -0,0 +1,28 @@
+<svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
why is there a sqlDatabaseProjects.png and an .svg?

Reply:
I just added everything Raden gave me.  The PNG is used by the extension gallery, dunno about current plans for the SVG, but it at least gives us the option to render higher-res raster images.